components/composer/src/Composer.svelte

```diff
@@ -369,6 +373,14 @@
           sendError = true;
         });
     } else if (id) {
+      //If message is an existing draft
+      if (msg.object === "draft") {
+        const res = await updateDraft(id, msg, access_token);
+        if (res.id) {
+          mergeMessage({ ...res });
+          msg = { draft_id: res.id, ...res };
```

Comment:
Use draft_id to tell `/send` api that we are sending an existing draft message.
No need to delete draft message at frontend code which api will handle it.  [send.py](https://github.com/nylas/cloud-core/blob/9d8f891fa3dc26133ba0661a764a3e5b3f806778/nylas/services/api/blueprints/send.py#L91)

Reply:
This is awesome!